src/styles/partials/components/_alt-list.scss

```diff
@@ -40,7 +40,7 @@ $icon-arrow-width: 32px;
     &:after {
       color: $gray-light;
       @include default-transition;
-      @include icon-content($icon-arrow-right);
+      @include icon-content($icon-arrow-right-circle);
```

Comment:
seems like you want to update the icon-arrow-right, instead of circle so it's update throughout?

Reply:
or is this only targeting links?

Reply:
@martypowell I don't think we want to do that, as icon-arrow-right variable is used for accordions currently. This just adds an updated icon to the alt-list component.

Reply:
to make it easier on the developers, can we rename this osmething maybe like `$icon-arrow-link` or something along those lines, something that might make a little more sense?

Reply:
Makes sense to me. Updated and committed.

Reply:
@martypowell Ready for a second look when you have a moment.